contextualized/analysis/pvals.py

```diff
@@ -90,24 +113,33 @@ def calc_homogeneous_context_effects_pvals(
 
 
 def calc_homogeneous_predictor_effects_pvals(
-    model: SKLearnWrapper, C: np.ndarray, **kwargs
+    model: SKLearnWrapper, C: np.ndarray, verbose: bool = True, **kwargs
 ) -> np.ndarray:
     """
     Calculate p-values for the context-invariant effects of predictors.
 
     Args:
         model (SKLearnWrapper): Model to analyze.
         C (np.ndarray): Contexts to analyze.
+        verbose (bool): Whether to print the range of possible p-values.
 
     Returns:
         np.ndarray: P-values of shape (n_predictors, n_outcomes) testing whether the
             sign of the context-invariant predictor effects are consistent across bootstraps.
     """
-    _, effects = get_homogeneous_predictor_effects(model, C, **kwargs)
-    # effects.shape: (n_predictors, n_bootstraps, n_outcomes)
-    if len(effects.shape) < 3:
+    if model.n_bootstraps < 2:
         print("P values are not well defined without multiple bootstrap samples.")
         return None
+    if verbose:
+        print(
+            "########################################################################################\n"
```

Comment:
prompt is shared with calc_homogeneous_predictor_effects_pvals, would prefer being defined in a helper function.